sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/service/metric/DistributionStatisticConfig.java

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (C) 2024-2024 Huawei Technologies Co., Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.sermant.core.service.metric;
+
+import java.time.Duration;
+
+/**
+ * DistributionStatisticConfig
+ *
+ * @author zwmagic
+ * @since 2024-08-17
+ */
+public class DistributionStatisticConfig {
+    private Boolean percentileHistogram;
+
+    private double[] percentiles;
+
+    private Integer percentilePrecision;
+
+    private Double minimumExpectedValue;
+
+    private Double maximumExpectedValue;
+
+    private Duration expiry;
```

Comment:
The name is inappropriate. It should be the validity period.

Reply:
these are same field definitions as micrmeter which are wrappered to use in Sermant